integration/tests/command_line/deploy_envs_without_modules.rb

```diff
@@ -0,0 +1,56 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'CODEMGMT-90 - C62419 - Deploy Environment without Module'
```

Comment:
This test doesn't make any sense as currently designed. We should re-design the test case.